velox/functions/prestosql/types/JsonType.cpp

```diff
@@ -15,6 +15,7 @@
  */
 
 #include "velox/functions/prestosql/types/JsonType.h"
+#include <type/DecimalUtil.h>
```

Comment:
Please, change to: #include "velox/type/DecimalUtil.h"

Reply:
done, thanks